packages/fast-components-react-base/src/toolbar/toolbar.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import { storiesOf } from "@storybook/react";
+import React from "react";
+import Toolbar from "./";
+import ToolbarItemGroup from "../toolbar-item-group";
+import Button from "../button";
+
+storiesOf("Toolbar", module)
+    .add("Default", () => (
+        <Toolbar>
+            <Button>Item 1</Button>
+            <button>Item 2</button>
+            <button>Item 3</button>
+            <ToolbarItemGroup>
+                <button>Item 4</button>
+                <button>Item 5</button>
+            </ToolbarItemGroup>
+        </Toolbar>
+    ))
+    .add("Empty", () => <Toolbar />)
+    .add("Ignore disabled elements", () => (
+        <Toolbar>
+            <button>Item 1</button>
+            <button disabled={true}>Disabled attribute</button>
+            <button disabled={false}>Item 3</button>
```

Comment:
iis `disabled={false}` necessary or just to explicitly show that this button is not disabled

Reply:
Just to make the example explicit and cover all cases.  Item one has no disabled attribute at all.